index.js

```diff
@@ -1,25 +1,177 @@
 const menuOptions = document.querySelectorAll('.mobile_menu-content a');
+function createCard(data) {
+  data.forEach((item) => {
+    const technologies = item.technologies
+      .map((technology) => `<li>${technology}</li>`)
+      .join('');
+    const card = document.createElement('div');
+    card.className = 'works-card';
+    card.innerHTML = `
+    <img
+      class="works-card--img"
+      src=${item.image}
+      alt="project pic"
+    />
+    <div class="works-card--desc">
+      <h2 class="works-card--title">${item.name}</h2>
+      <ul class="works-card--info">
+        <li>${item.info.company_name}</li>
+        <li>
+          <img src="images/works_images/Counter.png" alt="dot icon" />
+        </li>
+        <li>${item.info.position}</li>
+        <li>
+          <img src="images/works_images/Counter.png" alt="dot icon" />
+        </li>
+        <li>${item.info.date}</li>
+      </ul>
+      <p class="works-card--detail">
+        ${item.description}
+      </p>
+      <ul class="works-card--languages">
+         ${technologies}
+      </ul>
+      <button class="portfolio-button" type="button" onclick="openDetailWindow(${item.id})">See Project</button>
+    </div>
+    `;
+    document.getElementById('portfolio').appendChild(card);
+  });
+}
+function openDetailWindow(id) {
+  const openedDetail = projects.filter((item) => item.id === id.toString())[0];
+  const technologies = openedDetail.technologies
+    .map((technology) => `<li>${technology}</li>`)
+    .join('');
+  const detailContainer = document.querySelector('.detail-container');
+  detailContainer.innerHTML = `
+  <div>
+  <div class="detail-card">
+  <div class="upper-part">
+  <div class="first">
+    <h2 class="works-card--title">${openedDetail.name}</h2>
+    <button type="button" class="close-detail" onclick="closeDetail()">
+      <img src="images/closeDetail.png" alt="close icon" />
+    </button>
+  </div>
+  <ul class="works-card--info">
+  <li>${openedDetail.info.company_name}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.position}</li>
+  <li>
+    <img src="images/works_images/Counter.png" alt="dot icon" />
+  </li>
+  <li>${openedDetail.info.date}</li>
+  </ul>
+</div>
+<img src=${openedDetail.image} alt="project pic" />
+<div class="bottom-part">
+  <p class="works-card--detail">
+    ${openedDetail.description}
+  </p>
+  <div>
+    <ul class="works-card--languages">
+      ${technologies}
+    </ul>
+    <hr />
+    <div class="buttons-container">
+      <a class="portfolio-button" href=${openedDetail.live}>
+        <span>See live</span
+        ><img src="images/live_icon.png" alt="live icon" />
+      </a>
+      <a class="portfolio-button" href=${openedDetail.source}>
+        <span>See source</span
+        ><img src="images/source_icon.png" alt="source icon" />
+      </a>
+    </div>
+  </div>
+</div>
+</div>
+<div class="home_indicator"></div>
+</div>
+</div>`;
```

Comment:
Great job implementing the project popup window 👍However, I think it would be better to make the project cards and it's content fully responsive while switching from mobile version to desktop version. Right now, the project cards are not behaving according to figma template. For more clarity please see below 🔽

- [ ] All project cards are have different length and style on mobile version. For example, right now, your first popup window's length is different from the rest. Let's make sure that all project popup windows have same style for a better user experience. We don't want our user to have a bad experience? Right 🎯 Please see the video below for more detail 🔽

https://user-images.githubusercontent.com/106119105/216144705-ac200840-69fc-463f-a7b2-cbc55c6487af.mp4

### **_Mobile version 👇_**
- [ ] I think it would be better if you add a little bit more space between the list of technologies and buttons for mobile menu to follow the figma template. What do you think ?🤔
- [ ] Also what do you think of adding the same text for popup as given in figma template? 🤔🥇 If you see the text on project's card is less than the text present on popup window according to figma template. The purpose of the details window is to show more details of our project to the user. Right? Please make sure to add the same text as present in figma template.
- [ ] I think it would be better if you add the space below the project popup window. Right now, there's no space present. Let's make sure to add this one as well. I know you can so that 💪

| Your Design | Figma Design | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/106119105/216145859-753eda73-dd60-4b6c-b825-93dd9cbf13d8.png) | ![image](https://user-images.githubusercontent.com/106119105/216142014-4d92d9ad-ccfb-42dd-aef6-7b326e5bac39.png) | 

### **_Desktop version 👇_**
**_Please note that your project popup window looks okay on 50% zoom ratio but for 100% zoom ratio, it is missing some styles. I know you can fix this 💪_**
- [ ] Please make sure that buttons for desktop version are fully responsive while switching from mobile version to desktop version. Right now, from `768px` to `1192px` the buttons are not according to figma template. 
- [ ] Also what do you think of adding the same text for popup as given in figma template? 🤔🥇 If you see the text on project's card is less than the text present on popup window according to figma template. The purpose of the details window is to show more details of our project to the user. Right? Please make sure to add the same text as present in figma template.
- [ ]  I think it would be better if you add a little bit more space between the list of technologies and buttons for mobile menu to follow the figma template. What do you think ?🤔
- [ ] As suggested earlier, I think it would be better if you fix the spacing for the popup window's desktop version according to the figma template as well. Right now you have very less spacing. Also, please make sure to add the space below the project popup window. Right now, there's no space present. Let's make sure to add this one as well. I know you can so that 💪

| Your Design | Figma Design | 
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/106119105/216154668-97d5332e-d55c-46e0-a8b0-c1809bb3e0c3.png) | ![image](https://user-images.githubusercontent.com/106119105/216147334-72ca32d5-efb4-49ef-a3fe-ab1cdb97fb91.png) |